pkgs/tools/misc/graylog/plugins.nix

```diff
@@ -35,6 +35,19 @@ in {
       description = "SSO support for Graylog through trusted HTTP headers set by load balancers or authentication proxies";
     };
   };
+  internal-logs = glPlugin rec {
+    name = "graylog-internal-logs-${version}";
+    pluginName = "graylog-plugin-internal-logs";
+    version = "1.0.0";
+    src = fetchurl {
```

Comment:
@FRidh I do not know how I could use fetchFromGitHub to fetch files released on GitHub. Is it possible to fetch the released files (not the source)?